test/module/shared_model/interface_mocks.hpp

```diff
@@ -110,6 +127,27 @@ auto createMockBatchWithHash(
   return res;
 }
 
+/**
+ * Creates mock batch with provided transactions
+ * @param txs -- const ref to hash to be returned by the batch
```

Comment:
🙈  txs is not a hash.
I suggest to rewrite docstring with one line such as "Creates mock batch with provided transactions and reduced hash".